src/ui/flutter_app/lib/screens/game_page/game_page.dart

```diff
@@ -260,13 +205,7 @@ class _GamePageState extends State<GamePage>
             break;
           } else {
             debugPrint("[tap] putPiece: skip [$sq]");
-            if (mounted) {
-              showTip(S.of(context).tipBanPlace);
-              if (LocalDatabaseService.preferences.screenReaderSupport) {
-                ScaffoldMessenger.of(context).clearSnackBars();
```

Comment:
As mentioned in the message above, not all places need to call clearSnackBars(). If you want to refactor, you may need to pass an additional parameter.